frontend/src/graph/node.jsx

```diff
@@ -18,12 +18,14 @@ import {
 import classNames from "classnames";
 import textFit from "textfit";
 import { GrNew } from "react-icons/gr";
+import { FiDatabase } from "react-icons/fi";
+
 import { NotificationContext } from "../notifications/context";
 import { GraphContext } from "./graph-context";
 
 const BaseNode = ({ data, isNewNode }) => {
     const graph = useReactFlow();
-    const { nodeType, label } = data;
+    const { nodeType, label, hasCache, isActive } = data;
```

Comment:
added this 2 to add a badge on nodes that have cache and dashed border for nodes that are not active